lib/Middleware/Attribute/RequireDocumentSessionUserOrShareToken.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace OCA\Text\Middleware\Attribute;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_METHOD)]
+class RequireDocumentSessionUserOrShareToken {
```

Comment:
Not sure how to name it differently but maybe to make it clear that it is session or user and not SesssionUser
```suggestion
class RequireDocumentSessionOrUserOrShareToken {
```

Reply:
Agreed. Changed now.